package.json

```diff
@@ -35,6 +35,7 @@
     "emotion": "^10.0.23",
     "emotion-server": "^10.0.17",
     "es6-promise": "^4.2.5",
+    "framer-motion": "^1.10.3",
```

Comment:
Ouff, can we do this without framer? It is a HUGE library and i think it'd be good to omit it if we can. Alternatively, if we can split the code and load it asynchronously so that we don't include it in the initial bundle.

Reply:
Absolutely. I just took all the shortcuts I could with this but now that we have som extra time we can look at a more lightweight solution 🙌

Reply:
Removed it and tried to use react-spring. Updated that lib only to run in to some type and build errors. So resorted to plain css transitions for now, works fine I think 👍